python/cs_GUI.enaml

```diff
@@ -323,7 +323,7 @@ enamldef EvalProp(GroupBox):
         style_class << 'valid' if prop.valid else 'invalid'
 
     Label: val:
-        text << prop.valueStr
+        text << prop.valueStr if prop.showlabel else ''
```

Comment:
This seems like it could be useful but out of an abundance of caution I'd like to know why we need it.